test/integration/controllers/suite_test.go

```diff
@@ -66,9 +65,7 @@ type SelfSigner interface {
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
 
-	RunSpecsWithDefaultAndCustomReporters(t,
-		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}})
+	RunSpecs(t, "Controller Suite")
```

Comment:
@yue9944882  Hello, Kim, here we remove the custom reporter as ginkgo migrate guide suggest, if a junit-report is needed for you we can use `./e2e.test -test.v -ginkgo.v --ginkgo.junit-report <junit-filename>` to do it.